### PR TITLE
Move heatmap generation into repo

### PR DESCRIPTION
**Pull request by agraham1988**

### CODE REVIEW

File: heatmapGenerator/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# ais-uk.gov.ukho.ais.heatmaps
+[![Build Status](https://ukhogov.visualstudio.com/Pipelines/_apis/build/status/UKHO.ais-heatmap-mob?branchName=master)](https://ukhogov.visualstudio.com/Pipelines/_build/latest?definitionId=119&branchName=master)
```
Comment (JoelClemence) [inline heatmapGenerator/README.md]: Change badge to be right repo

File: heatmapGenerator/azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+variables:
```
Comment (JoelClemence) [inline heatmapGenerator/azure-pipelines.yml]: Is this still needed?

File: heatmapGenerator/build.gradle.kts [hunk 3]
```diff
@@ -35,18 +29,10 @@ dependencies {
...
-    testImplementation(sparkCoreDependency)
-    testImplementation(sparkSqlDependency)
-    testImplementation("junit:junit:${Versions.junit}")
-    testImplementation("org.assertj:assertj-core:${Versions.assertJ}")
 }
 
 application {
-    mainClassName = "uk.gov.ukho.ais.rasters.AisToRaster"
+    mainClassName = "uk.gov.ukho.ais.heatmaps.Main"
     applicationDefaultJvmArgs = listOf("-Dspark.master=local[*]")
```
Comment (galexite) [inline heatmapGenerator/build.gradle.kts]: Obsolete

File: heatmapGenerator/src/main/scala/uk/gov/ukho/ais/heatmaps/ConfigParser.scala [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+object ConfigParser {
+
+  @transient private final val PARSER = {
+    val builder = OParser.builder[Config]
+
+    def validatePositive: AnyVal => Either[String, Unit] = {
+      case value: Double if value > 0 => builder.success
+      case value: Int if value > 0    => builder.success
+      case value: Long if value > 0   => builder.success
+      case _                          => builder.failure("Negative values are not allowed")
```
Comment (galexite) [inline heatmapGenerator/src/main/scala/uk/gov/ukho/ais/heatmaps/ConfigParser.scala]: Should be `"Value must be greater than 0"` instead

**Pull request merged**